packages/api/lib/pdrHelpers.js

```diff
@@ -17,6 +18,61 @@ function generateShortPAN(disposition) {
   );
 }
 
+/**
+ * Get list of input granules from execution
+ *
+ * @param {string} executionArn - execution arn
+ * @returns {object[]} list of granules
+ */
+async function getGranulesFromExecution(executionArn) {
+  const excObj = await getExecution({
```

Comment:
Is the default retry on this acceptable for the use case?   Should this be denoted as an internal method for now?

Reply:
This method isn't being unit tested at all, via proxyquire.

Reply:
moved the logic of the method to generateLongPAN